specs/maas-v1.json

```diff
@@ -506,7 +651,7 @@
       }
     },
     "time": {
-      "description": "An UTC timestamp (number of milliseconds in a Date object since January 1, 1970, 00:00:00)",
+      "description": "A UTC timestamp (number of milliseconds in a Date object since January 1, 1970, 00:00:00)",
```

Comment:
FWIW, 'an' is correct here. The main reason for using 'a' vs. 'an' is about pronunciation. If there are two vowels in direct succession, 'an' makes it easier to pronounce.

Reply:
"UTC" is _pronounced_ with a "j" in the beginning, as in "you-tee-sea" => "a" is correct. And for instance "an API" instead of "a API".

Reply:
There isn't a hard rule here, and it was simply a note. There isn't a 'correct', in the British English sense of 'correctness'. 

For pragmatic reasons, it is easier to say 'an you-tee-sea' than 'a you-tee-see', so I recommend using 'an' here.